src/appkit.cs

```diff
@@ -11867,6 +11868,16 @@ partial interface NSResponder : NSCoding {
 		[Mac (10,12)]
 		[Export ("newWindowForTab:")]
 		void GetNewWindowForTab ([NullAllowed] NSObject sender);
+
+		[Export ("presentError:")]
+		bool PresentError (NSError error);
+
+		[Export ("willPresentError:")]
+		NSError WillPresentError (NSError error);
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)] // You probably shouldn't override this method, because you have no way of reliably predicting whether this method vs. -presentError will be invoked for any particular error.
```

Comment:
I agree on [Sealed].

I was tempted to make a less terrible version, but there are two other "copies" of PresentError in appkit.cs and I wasn't sure what's worse:

- Three identical (on different classes) versions of the same bad API.
- Two bad versions of the same API and one good one.

In either case, I can work on a XAMCORE_4_0 version for at least two, and do something (either match or be different) w\ this new one.